tests/test_registration.py

```diff
@@ -147,7 +148,8 @@ def test_landmark_registration(self):
         np.testing.assert_array_almost_equal(fd_reg(center), original_values,
                                              decimal=2)
 
-    def test_mse_decomposition(self):
+    def _test_mse_decomposition(self):
+        # Test disabled
```

Comment:
Should it not be tested with the new API?

Reply:
I wanted to wait for your review on how to do the API of the module before doing the tests.

Reply:
This test was updated in 695c74a. The new api is now tested too.